bundles/framework/announcements/view/AnnouncementsCollapse.jsx

```diff
@@ -18,29 +19,33 @@ const AnnouncementsCollapse = ({controller, checked, announcements, modals }) =>
               <AnnouncementsModal
               id={modal.id}
               controller={controller}
-              title={modal.title}
-              content={modal.content}
+              title={modal.locale[lang].name}
+              content={modal.locale[lang].content}
               key={modal.id}
               index={index}
               checked={checked}
               />
             );
           })}
       <div>
-      <Collapse accordion>
-          { announcements.map((announcement) => {
-            let start = moment(announcement.begin_date).format(DATEFORMAT);
-            let end = moment(announcement.end_date).format(DATEFORMAT);
-              return (
-                <CollapsePanel header={announcement.title} key={announcement.id}>
-                    <h3><b>{announcement.title}</b></h3>
-                    <p>{announcement.content}</p>
-                    <Divider />
-                    <b><Message messageKey={'valid'} /></b>
-                    <p>{start.toString()} - {end.toString()}</p>
-                </CollapsePanel>)
-          })}
-        </Collapse>
+        { announcements.length > 0 ?
+          <Collapse accordion>
+            { announcements.map((announcement) => {
+              let start = moment(announcement.begin_date).format(DATEFORMAT);
+              let end = moment(announcement.end_date).format(DATEFORMAT);
+                return announcement.locale[lang].name && (
```

Comment:
Would be nice to use Oskari.getLocalized() here as well, but we'll change it later.